bucket/MPC-Video-Renderer.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "0.6.7",
+    "description": "MPC Video Renderer is a free and open-source video renderer for DirectShow. ",
+    "homepage": "https://github.com/Aleksoid1978/VideoRenderer",
+    "license": "GPL-3.0",
+    "suggest": {
+        "Media Player Classic - Home Cinema fork": "extras/mpc-hc-fork"
+    },
+    "url": "https://github.com/Aleksoid1978/VideoRenderer/releases/download/0.6.7/MpcVideoRenderer-0.6.7.2013.zip",
+    "hash": "215D1F382A4A399052D7B3618C082BAC658DF8A03FF671471C028FC52DB5FCA9",
+    "pre_install": [
+        "if ($architecture -eq '64bit') { Start-Process -FilePath \"$dir\\Install_MPCVR_64.cmd\" -Verb RunAs }",
+        "elseif ($architecture -eq '32bit') { Start-Process -FilePath \"$dir\\Install_MPCVR_32.cmd\" -Verb RunAse }"
+    ],
+    "uninstaller": {
+        "script": [
+            "if ($architecture -eq '64bit') { Start-Process -FilePath \"$dir\\Uninstall_MPCVR_64.cmd\" -Verb RunAs }",
+            "elseif ($architecture -eq '32bit') { Start-Process -FilePath \"$dir\\Uninstall_MPCVR_32.cmd\" -Verb RunAse }"
+        ]
+    },
+    "checkver": {
+        "github": "https://github.com/Aleksoid1978/VideoRenderer"
+    },
+    "autoupdate": {
+        "url": "https://github.com/Aleksoid1978/VideoRenderer/releases/download/v$version/MpcVideoRenderer-$version.2013.zip"

```

Comment:
```suggestion
        "url": "https://github.com/Aleksoid1978/VideoRenderer/releases/download/$matchHead/MpcVideoRenderer-$version.zip"
```

Reply:
checkver not working because download link look like this
`https://github.com/Aleksoid1978/VideoRenderer/releases/download/0.6.7/MpcVideoRenderer-0.6.7.2013.zip`

so every new release came with new random 4 digit number `MpcVideoRenderer-$version.2013`
we have to setup regex for that and i did try to make regex for that
```
    "checkver": {
        "url": "https://github.com/Aleksoid1978/VideoRenderer",
        "regex": "/MpcVideoRenderer-([\\d.]+).(?<commit>\\d{4})"
    },
    "autoupdate": {
        "url": "https://github.com/Aleksoid1978/VideoRenderer/releases/download/$version/MpcVideoRenderer-$version.$matchCommit.zip"
    }
```
but this is not working also i don't know where is the issue 😢

Reply:
see next suggested change, I forgot to submit it.

Reply:
ooh okey thanks